### PR TITLE
Clean test for ERSSUP-57222

### PR DESCRIPTION
**Pull request by nhsd-jonathan-heather**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: manifest_template.yml [hunk 1]
```diff
@@ -27,9 +27,19 @@ APIGEE_ENVIRONMENTS:
...
 
 INTERNAL_DEV_VARIANTS:
-- alpha
-- rc
-- fix
+- name: alpha
+- name: rc
+- name: fix
+- name: ft01
+  service: fti
```
Comment (RajaGovindharaj16) [inline manifest_template.yml:34]: I assume FT is new environment and never had existing product in it. if existing products are there, need to migrate the apps.

**Review by RajaGovindharaj16 — COMMENTED** [review]
I just finished the review of mainfest-template.yaml

File: manifest_template.yml (hunk 1 above)
Comment (nhsd-jonathan-heather) [reply manifest_template.yml:34]: Hi Raja, we have a new "FT" app that supports access to the FT environments - this also sets out the correct ASID etc.

Comment (nhsd-jonathan-heather) [reply manifest_template.yml:34]: .. The FT environments have to be present in the manifest, but in fact - we only "use" them in the PR pipeline definition - so its for development only - and the PR pipeline creates unique products for the PR.

File: azure/azure-pr-pipeline.yml [hunk 2]
```diff
@@ -43,5 +42,40 @@ extends:
...
+      - environment: internal-dev
+        service_name: ${{ variables.service_name }}-fti
+        short_service_name: ${{ variables.short_service_name }}-fti
+        service_base_path: ${{ variables.service_base_path }}-ft01
+        stage_name: ers_ft01_pr
+        jinja_templates:
+          ERS_TARGET_SERVER: e-referrals-service-api--ft-8443
+      - environment: internal-dev
+        service_name: ${{ variables.service_name }}-ftiv
+        short_service_name: ${{ variables.short_service_name }}-ftiv
```
Comment (adamoldfield-nhs) [inline azure/azure-pr-pipeline.yml:54]: As discussed, not allowed numbers in the service names

File: proxies/live/apiproxy/targets/ers-target.xml [hunk 3]
```diff
@@ -89,6 +89,9 @@
   </PreFlow>
   <HTTPTargetConnection>
     <SSLInfo>
+{% if '--ft-' in (ERS_TARGET_SERVER | default('e-referrals-service-api')) %}
```
Comment (adamoldfield-nhs) [inline proxies/live/apiproxy/targets/ers-target.xml:92]: Overrides default trust store, just for FTs.

**Review by adamoldfield-nhs — APPROVED** [review]

**Pull request merged**